pom.xml

```diff
@@ -648,6 +653,10 @@
 							<goals>
 								<goal>jar</goal>
 							</goals>
+							<!-- avoid doclint warnings that make maven build fail in java 8 -->
```

Comment:
This should have been fixed already by #318 with a profile activated only for JDK >=8. This fix makes the travis build fail as it is run on Java 7. Could you rebase your pull request to include latest fixes ?